nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -94,14 +94,22 @@ let
     REDIRECT_STATUS   = "200";
   };
 
-  recommendedProxyConfig = pkgs.writeText "nginx-recommended-proxy-headers.conf" ''
+  recommendedProxyConfig = pkgs.writeText "nginx-recommended-proxy_set_header-headers.conf" ''
     proxy_set_header        Host $host;
     proxy_set_header        X-Real-IP $remote_addr;
     proxy_set_header        X-Forwarded-For $proxy_add_x_forwarded_for;
     proxy_set_header        X-Forwarded-Proto $scheme;
     proxy_set_header        X-Forwarded-Host $host;
     proxy_set_header        X-Forwarded-Server $host;
   '';
+  recommendedUwsgiConfig = pkgs.writeText "nginx-recommended-uwsgi_param-headers.conf" ''
+    uwsgi_param             HTTP_HOST $host;
+    uwsgi_param             HTTP_X_REAL_IP $remote_addr;
+    uwsgi_param             HTTP_X_FORWARDED_FOR $proxy_add_x_forwarded_for;
+    uwsgi_param             HTTP_X_FORWARDED_PROTO $scheme;
+    uwsgi_param             HTTP_X_FORWARDED_HOST $host;
+    uwsgi_param             HTTP_X_FORWARDED_SERVER $host;
```

Comment:
I'm a bit confused why these are needed and if anything even would use them

Reply:
uwsgi_params are generally used by uwsgi and/or its plugins, not directly accessed by applications. Some of the *very specific* use-cases are here https://uwsgi-docs.readthedocs.io/en/latest/Nginx.html

I can't say the above params are recommendedUwsgiConfig